src/app/timetable/services/time-table.service.ts

```diff
@@ -1,13 +1,19 @@
-import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { Subject } from "rxjs";
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: "root"
 })
 export class TimeTableService {
-  constructor(private http:HttpClient) { }
+  private messageSrc = new Subject<string>();
+  message = this.messageSrc.asObservable();
```

Comment:
1. Rename `message` in `message$` because is an Observable (a Subject is also an observable)
2. You can use `messageSrc` as `message`. It is save to remove `message` and use `messageSrc`. And also add an `$` at the end of `messageSrc`